clearwater-diags-monitor/usr/share/clearwater/bin/gather_diags_utils

```diff
@@ -0,0 +1,95 @@
+#!/bin/sh
```

Comment:
- Why is this a separate script?
- Why is it separately executable?
- Why is it in the /bin folder? 

Shouldn't it just be part of the main gather diags process?

Reply:
* Because previously i was using the functions in both files
* Because? Wasn't really thinking about that tbh, probably should have. Can just remove the binsh
* Because it was simplest to have it alongside everything else